dokka-subprojects/plugin-base/src/main/resources/dokka/templates/includes/header.ftl

```diff
@@ -22,10 +22,10 @@
     </div>
     <div class="navigation-controls">
         <#if homepageLink?has_content>
-            <div class="navigation-controls--btn navigation-controls--homepage" id="homepage-link" role="button"><a href="${homepageLink}"></a></div>
+            <a class="navigation-controls--button navigation-controls--button_homepage" id="homepage-link" href="${homepageLink}" ></a>
```

Comment:
@IgnatBeresnev I've just found out about header/footer html templates being used in user's projects and about corresponding problems caused by changes in html. So how bad is this change and should I revert it maybe for this time and butch all such changes to a special release?

Reply:
Good thought! The homepage button specifically hasn't been released yet, so it's perfectly fine and expected to change the styles related to it, see https://github.com/Kotlin/dokka/pull/3235/files#diff-316e3e9b3d50ff73dcc848844e5eaac72b2d38b7f1e84dca8a383a8b2c5a162c

As for other "unnecessary" / refactoring changes, I think it makes sense to minimize them in this release (1.9.20). We can break it down into two PRs: the first one fixes the issue in the most compatible way, we'll merge it and then have the code freeze at the end of the week. The second one introduces some additional changes from this PR, we'll merge it after code freeze

Reply:
Sounds good, I've rolled back unnecessary changes to the template